app/namespaces/test_adjustment/test_adjustment_controller.py

```diff
@@ -0,0 +1,66 @@
+import logging.config
+
+from flask_restplus import Resource, Namespace
+from flask import jsonify, make_response, request
+
+from app.utils import validate_request_input_against_schema
+from .test_adjustment_schema import TestAdjustmentSchema
+from .test_adjustment_service import TestAdjHandler
+
+
+test_adjustment_ns = Namespace('test_adjustment', description='Endpoints for updating airtable records via scripting.')
+logging.getLogger(__name__)
+
+
+@test_adjustment_ns.route('/get_adjusted', methods=['POST'])
+class Records(Resource):
+    @test_adjustment_ns.doc('An endpoint for getting all records from database with or without filters.')
```

Comment:
Is this the right documentation for this endpoint? Isn't this a test adj endpoint?